torchdata/datapipes/iter/load/fsspec.py

```diff
@@ -126,17 +127,20 @@ class FSSpecFileOpenerIterDataPipe(IterDataPipe[Tuple[str, StreamWrapper]]):
         >>> file_dp = datapipe.open_files_by_fsspec()
     """
 
-    def __init__(self, source_datapipe: IterDataPipe[str], mode: str = "r", **kwargs) -> None:
+    def __init__(
+        self, source_datapipe: IterDataPipe[str], mode: str = "r", kwargs_for_open: Optional[Dict] = None, **kwargs
```

Comment:
nit: Could we make `kwargs_for_open` as a keyword argument?
```
def __init__(
        self, source_datapipe: IterDataPipe[str], mode: str = "r", *, kwargs_for_open: Optional[Dict] = None, **kwargs
```
Using it as a positional argument might make extending those DataPipes with more positional arguments hard in the future.